.circleci/config.yml

```diff
@@ -932,56 +880,78 @@ jobs:
       resource_class:
         description: Machine resource class
         type: string
-        default: ethereum-optimism/latitude-1
-      skip_slow_tests:
-        description: Indicates that slow tests should be skipped
-        type: boolean
-        default: false
+        default: ethereum-optimism/latitude-1-go-e2e
       no_output_timeout:
         description: Timeout for running tests
         type: string
         default: 5m
+      environment_overrides:
+        description: Environment overrides
+        type: string
+        default: ""
+      packages:
+        description: List of packages to test
+        type: string
+        default: |
```

Comment:
shouldn't the default be empty, and this list of packages provided to the job instance below?
This way the granularity of execution would be more clearly the responsibility of the downstream pipeline

Reply:
No - we need the ability to only specify a subset of the jobs in order to let the fault proof jobs run only a specific package.

Reply:
I suppose we can specify them all in the job itself but then any other e2e job is going to have to paste in a huge list as well.

Reply:
Upon further reflection I like your idea here - updated to pass in the packages from the job instance.